.github/workflows/main.yml

```
@@ -22,3 +23,12 @@ jobs:
       run: npm publish
       env:
         NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
```

```suggestion
        node-version: '13.x'
```
Maybe just to match the other one

---

yep, changed it. good call out.